### PR TITLE
DOCS-2277 Browser Monitoring Edits

### PR DESCRIPTION
**Pull request by alai97**
## Changes

Updates SKU naming for the Browser Premium plan and adds information for RUM services.

## Motivation

DOCS-2277

### Preview

https://docs-staging.datadoghq.com/alai97/rum-services-doc-updates/real_user_monitoring/browser/

## Testing

- [x] Local
- [x] Staging
- [ ] Unit
- [ ] End to end

---

I have gone over the [contributing](https://github.com/DataDog/browser-sdk/blob/main/CONTRIBUTING.md) documentation.

### CODE REVIEW

File: packages/rum/README.md [hunk 1]
```diff
@@ -321,6 +325,15 @@ datadogRum.init({
...
 ```
 
+### Tagging
+
+A service is an independent, deployable code repository that maps to a set of pages. 
+
+- If your browser application was constructed as a monolith, your RUM application has one service name for the application. 
+- If your browser application was constructed as separate repositories for multiple pages, edit the default service names throughout the lifecycle of your application. 
+
+For more information, see Configuring Browser Services (link here).
```
Comment (alai97) [inline packages/rum/README.md:335]: Link TBD here @amina196

File: packages/rum/README.md [hunk 2]
```diff
@@ -277,19 +281,19 @@ init(configuration: {
...
 
 This feature requires the RUM Browser SDK v3.6.0+.
 
-When a new session is created, it can be tracked as either:
+When a session is created, RUM tracks it as either:
 
-- [**Browser RUM**][11]: Only _Sessions_, _Views_, _Actions_, and _Errors_ are collected. Calls to `startSessionReplayRecording()` are ignored.
-- [**RUM Premium**][11]: Everything from Browser RUM, including _Resources_, _Long Tasks_, and _Replay_ recordings are collected. To collect _Replay_ recordings, call `startSessionReplayRecording()`.
+- [**Browser RUM**][11]: Only sessions, views, actions, and errors are collected. Calls to `startSessionReplayRecording()` are ignored.
+- [**Browser Premium**][11]: Everything from Browser RUM, including resources, long tasks, and replay recordings, is collected. To collect replay recordings, call `startSessionReplayRecording()`.
```
Comment (buraizu) [inline packages/rum/README.md:291]:
```suggestion
- [**Browser Premium**][11]: Everything from Browser RUM is collected, including resources, long tasks, and replay recordings. To collect replay recordings, call `startSessionReplayRecording()`.
```

**Review by buraizu — APPROVED** [review]
Thanks for correcting that minor grammar issue! With that said, now that the parentheses are gone, I think this edit makes the sentence a bit more clear.

File: packages/rum/README.md [hunk 3]
```diff
@@ -337,9 +350,12 @@ datadogRum.init({
...
-[11]: https://www.datadoghq.com/pricing/?product=real-user-monitoring#real-user-monitoring
+[11]: https://www.datadoghq.com/pricing/?product=real-user-monitoring--session-replay#real-user-monitoring--session-replay
 [12]: https://docs.datadoghq.com/real_user_monitoring/connect_rum_and_traces?tab=browserrum
 [13]: https://docs.datadoghq.com/real_user_monitoring/session_replay/privacy_options?tab=maskuserinput
 [14]: https://docs.datadoghq.com/agent/basic_agent_usage#datadog-site
 [15]: https://docs.datadoghq.com/getting_started/tagging/#defining-tags
 [16]: https://docs.datadoghq.com/real_user_monitoring/browser/monitoring_page_performance/#how-page-activity-is-calculated
+[17]: https://docs.datadoghq.com/real_user_monitoring/session_replay/
+[18]: https://docs.datadoghq.com/real_user_monitoring/session_replay/privacy_options
+[19]: https://docs.datadoghq.com/getting_started/tagging/using_tags
```
Comment (alai97) [inline packages/rum/README.md:361]:
```suggestion
[19]: https://docs.datadoghq.com/getting_started/tagging/using_tags/#rum--session-replay
```

File: packages/rum/README.md [hunk 4]
```diff
@@ -2,21 +2,25 @@
...
-To set up RUM Browser Monitoring:
+To set up RUM Browser Monitoring, create a RUM application:
 
 1. In Datadog, navigate to the [**RUM Applications** page][1] and click the **New Application** button.
 2. Enter a name for your application and click **Generate Client Token**. This generates a `clientToken` and an `applicationId` for your application.
-3. Setup the RUM Browser SDK with [npm](#npm) or a hosted version ([CDN async](#cdn-async) or [CDN sync](#cdn-sync)).
-4. Deploy the changes to your application. Once your deployment is live, Datadog collects events from your users' browsers.
-5. Visualize the [data collected][2] in [dashboards][3] or create a search query in the [RUM Explorer][16].
+3. Choose the instrumentation type for the RUM Browser SDK: [npm](#npm), or a hosted version ([CDN async](#cdn-async) or [CDN sync](#cdn-sync)).
+4. Define the environment name, service name, and deployed version for your application to use [Unified Service Tagging][19] for RUM & Session Replay. For more information, see [Tagging](#tagging).
```
Comment (alai97) [inline packages/rum/README.md:16]:
```suggestion
4. Define the environment name, service name, and deployed version for your application to use unified service tagging for [RUM & Session Replay][19]. For more information, see [Tagging](#tagging).
```

File: packages/rum/README.md [hunk 5]
```diff
@@ -2,21 +2,25 @@
...
+5. Set the sampling rate of total user sessions collected and use the slider to set the percentage of total [Browser Premium][11] sessions collected. Browser Premium sessions include resources, long tasks, and replay recordings.
+6. Click the **Session Replay Enabled** toggle to enable users with permissions to access [Session Replay][17].
```
[the remark this answers is not in the document]
Comment (alai97) [reply packages/rum/README.md:18]:
```suggestion
6. Click the **Session Replay Enabled** toggle to enable [Session Replay][17] to record user sessions.
```

File: packages/rum/README.md [hunk 6]
```diff
@@ -2,21 +2,25 @@
...
 
-To set up RUM Browser Monitoring:
+To set up RUM Browser Monitoring, create a RUM application:
 
 1. In Datadog, navigate to the [**RUM Applications** page][1] and click the **New Application** button.
 2. Enter a name for your application and click **Generate Client Token**. This generates a `clientToken` and an `applicationId` for your application.
-3. Setup the RUM Browser SDK with [npm](#npm) or a hosted version ([CDN async](#cdn-async) or [CDN sync](#cdn-sync)).
-4. Deploy the changes to your application. Once your deployment is live, Datadog collects events from your users' browsers.
-5. Visualize the [data collected][2] in [dashboards][3] or create a search query in the [RUM Explorer][16].
+3. Choose the instrumentation type for the RUM Browser SDK: [npm](#npm), or a hosted version ([CDN async](#cdn-async) or [CDN sync](#cdn-sync)).
```
[the remark this answers is not in the document]
Comment (alai97) [reply packages/rum/README.md:15]:
```suggestion
3. Choose the installation type for the RUM Browser SDK: [npm](#npm), or a hosted version ([CDN async](#cdn-async) or [CDN sync](#cdn-sync)).
```

File: packages/rum/README.md [hunk 7]
```diff
@@ -311,7 +315,7 @@ datadogRum.init({
 });
 ```
 
-For example, to track only 50% of your overall sessions with half tracked as Browser RUM and the other half tracked as RUM Premium, set the `sampleRate` and the `premiumSampleRate` to 50:
+For example, to track only 50% of your overall sessions with half tracked as Browser RUM and the other half tracked as Browser Premium, set the `sampleRate` and the `premiumSampleRate` to 50:
```
Comment (alai97) [inline packages/rum/README.md:318]:
```suggestion
For example, to track only 50% of your overall sessions with half tracked as Browser and the other half tracked as Browser Premium, set the `sampleRate` and the `premiumSampleRate` to 50. 25% of your sessions are Browser Premium. 
```

File: packages/rum/README.md [hunk 8]
```diff
@@ -277,19 +281,19 @@ init(configuration: {
...
 
 This feature requires the RUM Browser SDK v3.6.0+.
 
-When a new session is created, it can be tracked as either:
+When a session is created, RUM tracks it as either:
 
-- [**Browser RUM**][11]: Only _Sessions_, _Views_, _Actions_, and _Errors_ are collected. Calls to `startSessionReplayRecording()` are ignored.
-- [**RUM Premium**][11]: Everything from Browser RUM, including _Resources_, _Long Tasks_, and _Replay_ recordings are collected. To collect _Replay_ recordings, call `startSessionReplayRecording()`.
+- [**Browser RUM**][11]: Only sessions, views, actions, and errors are collected. Calls to `startSessionReplayRecording()` are ignored.
+- [**Browser Premium**][11]: Everything from Browser RUM is collected, including resources, long tasks, and replay recordings. To collect replay recordings, call `startSessionReplayRecording()`.
```
Comment (alai97) [inline packages/rum/README.md:291]:
```suggestion
- [**Browser Premium**][11]: Everything from Browser is collected, including resources, long tasks, and replay recordings. To collect replay recordings, call `startSessionReplayRecording()`.
```

File: packages/rum/README.md [hunk 9]
```diff
@@ -277,19 +281,19 @@ init(configuration: {
 })
 ```
 
-### Browser RUM and RUM Premium sampling configuration
+### Browser RUM and Browser Premium sampling configuration
```
[the remark this answers is not in the document]
Comment (alai97) [reply packages/rum/README.md:284]:
```suggestion
### Browser and Browser Premium sampling configuration
```

File: packages/rum/README.md [hunk 10]
```diff
@@ -200,7 +204,7 @@ See `premiumSampleRate`.
 : Optional<br/>
 **Type**: Number<br/>
 **Default**: `100`<br/>
-The percentage of tracked sessions with [Premium pricing][11] features: `100` for all, `0` for none. For more details about `premiumSampleRate`, see the [sampling configuration](#browser-rum-and-rum-premium-sampling-configuration).
+The percentage of tracked sessions with [Browser Premium pricing][11] features: `100` for all, `0` for none. For more details about `premiumSampleRate`, see the [sampling configuration](#browser-rum-and-rum-premium-sampling-configuration).
```
Comment (alai97) [inline packages/rum/README.md:207]:
```suggestion
The percentage of tracked sessions with [Browser Premium pricing][11] features: `100` for all, `0` for none. For more details about `premiumSampleRate`, see the [sampling configuration](#browser-and-rum-premium-sampling-configuration).
```

File: packages/rum/README.md [hunk 11]
```diff
@@ -277,19 +281,19 @@ init(configuration: {
...
+- [**Browser RUM**][11]: Only sessions, views, actions, and errors are collected. Calls to `startSessionReplayRecording()` are ignored.
+- [**Browser Premium**][11]: Everything from Browser RUM is collected, including resources, long tasks, and replay recordings. To collect replay recordings, call `startSessionReplayRecording()`.
 
 Two initialization parameters are available to control how the session is tracked:
 
 - `sampleRate` controls the percentage of overall sessions being tracked. It defaults to `100%`, so every sessions is tracked by default.
-- `premiumSampleRate` is applied **after** the overall sample rate, and controls the percentage of sessions tracked as RUM Premium. It defaults to `100%`, so every sessions is tracked as RUM Premium by default.
+- `premiumSampleRate` is applied **after** the overall sample rate, and controls the percentage of sessions tracked as Browser Premium. It defaults to `100%`, so every sessions is tracked as Browser Premium by default.
 
 To track 100% of your sessions as Browser RUM:
```
Comment (alai97) [inline packages/rum/README.md:298]:
```suggestion
To track 100% of your sessions as Browser:
```

File: packages/rum/README.md [hunk 12]
```diff
@@ -2,21 +2,25 @@
...
 
-Datadog Real User Monitoring (RUM) enables you to visualize and analyze the real-time performance and user journeys of your application's individual users.
+Datadog Real User Monitoring (RUM) enables you to visualize and analyze the real-time performance and user journeys of your application's individual users. To collect events, add the RUM Browser SDK to your browser application and configure what data is collected using initialization parameters.
 
 ## Setup
 
 The RUM Browser SDK supports all modern desktop and mobile browsers including IE11. For more information, see the [Browser Support][8] table.
 
-To set up RUM Browser Monitoring:
+To set up RUM Browser Monitoring, create a RUM application:
```
[the remark this answers is not in the document]
Comment (alai97) [reply packages/rum/README.md:11]: That's fair, but we are encouraging users to set the environment and service names as part of the Instrument your application step, and if we are documenting one part, I feel like we should document it all.

File: packages/rum/README.md [hunk 13]
```diff
@@ -311,16 +315,27 @@ datadogRum.init({
 });
 ```
 
-For example, to track only 50% of your overall sessions with half tracked as Browser RUM and the other half tracked as RUM Premium, set the `sampleRate` and the `premiumSampleRate` to 50:
+For example, to track 60% of overall user sessions and set 70% of those sessions as Browser Premium, set `sampleRate` to 60 and `premiumSampleRate` to 70.
```
Comment (alai97) [inline packages/rum/README.md:318]:
```suggestion
The `premiumSampleRate` is a percentage of `sampleRate`. If you set `sampleRate` to 60 and `premiumSampleRate` to 50, 40% of sessions are dropped, 30% of sessions are collected as Browser and 30% of sessions are collected as Browser Premium. 
```

File: packages/rum/README.md [hunk 14]
```diff
@@ -311,16 +315,27 @@ datadogRum.init({
...
-For example, to track only 50% of your overall sessions with half tracked as Browser RUM and the other half tracked as RUM Premium, set the `sampleRate` and the `premiumSampleRate` to 50:
+For example, to track 60% of overall user sessions and set 70% of those sessions as Browser Premium, set `sampleRate` to 60 and `premiumSampleRate` to 70. 
 
 ```
 datadogRum.init({
     ....
-    sampleRate: 50,
-    premiumSampleRate: 50
+    sampleRate: 60,
+    premiumSampleRate: 70
```
Comment (alai97) [inline packages/rum/README.md:324]:
```suggestion
    premiumSampleRate: 50
```

File: packages/rum/README.md [hunk 15]
```diff
@@ -311,16 +315,27 @@ datadogRum.init({
...
 });
 ```
 
+Within the 60% of user sessions collected for your application, 18% are Browser and 42% are Browser Premium.
```
Comment (alai97) [inline packages/rum/README.md:328]:
```suggestion
```

File: packages/rum/README.md [hunk 16]
```diff
@@ -311,16 +315,26 @@ datadogRum.init({
 });
 ```
 
-For example, to track only 50% of your overall sessions with half tracked as Browser RUM and the other half tracked as RUM Premium, set the `sampleRate` and the `premiumSampleRate` to 50:
+The `premiumSampleRate` is a percentage of `sampleRate`. If you set `sampleRate` to 60 and `premiumSampleRate` to 50, 40% of sessions are dropped, 30% of sessions are collected as Browser and 30% of sessions are collected as Browser Premium.
```
Comment (BenoitZugmeyer) [inline packages/rum/README.md:318]: **👏 praise:** ‏thanks, that is clearer!

Comment (alai97) [reply packages/rum/README.md:318]: Thanks @nicholas-devlin and @MirandaKapin for initiating the feedback!

File: packages/rum/README.md [hunk 17]
```diff
@@ -277,21 +281,21 @@ init(configuration: {
 })
 ```
 
-### Browser RUM and RUM Premium sampling configuration
+### Browser and Browser Premium sampling configuration
 
 This feature requires the RUM Browser SDK v3.6.0+.
```
Comment (alai97) [inline packages/rum/README.md:286]:
```suggestion
This feature requires the RUM Browser SDK v3.0.0+. 

<div class="alert alert-info">
The RUM Browser SDK v4.10.2 introduces the `premiumSampleRate` initialization parameter, deprecating the `replaySampleRate` initialization parameter.
</div>
```

**Review by BenoitZugmeyer — CHANGES_REQUESTED** [review]
The link to the proxy guide leads to a 404.

**Review by BenoitZugmeyer — APPROVED** [review]

**Pull request merged**